l10n_ar_afipws/wizard/res_partner_update_from_padron_wizard_view.xml

```diff
@@ -12,43 +12,43 @@
                     </group>
-                    <p attrs="{'invisible': [('state', '!=', ('option'))]}" class="oe_grey">Only Partners with cuit are going to be updated.
+                    <p invisible="state != 'option'" class="oe_grey">Only Partners with cuit are going to be updated.
                         <br/>
                         Select the list of fields you want to update.</p>
-                    <group attrs="{'invisible': [('state', 'not in', ('option',))]}">
+                    <group invisible="state != 'option'">
                         <field name="field_to_update_ids" options="{'no_create': True}" widget="many2many_tags"/>
                         <field name="update_constancia"/>
                         <field name="title_case"/>
                     </group>
-                    <group attrs="{'invisible': [('state', 'in', ('option', 'finished'))]}" col="1">
+                    <group invisible="state != 'option'" col="1">
                         <h1>
-                            <field attrs="{'required': [('state', '=', 'selection')]}" name="partner_id" options="{'no_open': True}"/>
+                            <field required="state == 'selection'" name="partner_id" options="{'no_open': True}"/>
                         </h1>
                         <span>
                             <field name="field_ids" nolabel="1">
                                 <tree create="false" editable="top">
                                     <field name="field"/>
-                                    <field attrs="{'readonly':True}" name="old_value"/>
+                                    <field name="old_value" readonly="1"/>
                                     <field name="new_value"/>
                                 </tree>
                             </field>
                         </span>
                     </group>
                 </sheet>
                 <footer>
-                    <button attrs="{'invisible': [('state', 'in', ('option', 'finished' ))]}" class="oe_highlight" name="update_selection" string="Update Selection" type="object"/>
-                    <button attrs="{'invisible': [('state', '!=', 'selection')]}" class="oe_link" name="next_cb" string="Skip these partner" type="object"/>
-                    <button attrs="{'invisible': [('state', '!=', 'option')]}" class="oe_highlight" name="start_process_cb" string="Update with Manual Check" type="object"/>
-                    <button attrs="{'invisible': [('state', '!=', 'option')]}" class="oe_highlight" confirm="Are you sure to execute the automatic update of your partners ?" name="automatic_process_cb" string="Update Automatically" type="object"/>
-                    <span attrs="{'invisible': [('state', '=', 'finished')]} " class="or_cancel">or
+                    <button invisible="state not in ('option', 'finished')" class="oe_highlight" name="update_selection" string="Update Selection" type="object"/>
```

Comment:
La condición quedo al revés de como era es correcto esto? 

Sino puede quedar asi
`<button invisible="state not in ('option', 'finished')" class="oe_highlight" name="update_selection" string="Update Selection" type="object"/>`